num_palindrom.py

```diff
@@ -4,11 +4,14 @@ def highest_value_palindrome(palindrome_string, steps):
         return palindrome_string
     steps-=1
     numbers, count, sol= [], [],[]
+    
     for num in palindrome_string:
         if not int(num) in numbers:
             numbers.append(int(num))
             count.append(palindrome_string.count(num))
+            
     single = [1 for x in count if x%2!=0]
+    
     if len(palindrome_string)%2 ==0:
```

Comment:
zamiast pisać porównanie możesz sprawdzić czy reszta istnieje czy nie :)

```suggestion
    if not len(palindrome_string)%2:
```